### PR TITLE
Rename MASTER_EXTERNAL_PORT to COORDINATOR_EXTERNAL_PORT

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

**Review by gurkanindibay — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   master:
```
Comment (hanefi) [inline docker-compose.yml:4]: `s/master/coordinator/`

File: docker-compose.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
...
     container_name: "${COMPOSE_PROJECT_NAME:-citus}_master"
```
Comment (hanefi) [inline docker-compose.yml:5]: `s/master/coordinator/`

File: docker-compose.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ services:
...
     image: 'citusdata/citus:9.5.2'
-    ports: ["${MASTER_EXTERNAL_PORT:-5432}:5432"]
+    ports: ["${COORDINATOR_EXTERNAL_PORT:-5432}:5432"]
     labels: ['com.citusdata.role=Master']
```
Comment (hanefi) [inline docker-compose.yml:8]: `s/master/coordinator/`

I'm unaware how labels are used here.

File: nightly/docker-compose.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ services:
   master:
```
Comment (hanefi) [inline nightly/docker-compose.yml:4]: `s/master/coordinator/`

**Review by hanefi — COMMENTED** [review]
There are many more places where the word master is used. I think we should understand the implications and rename all occurences of said word.

File: nightly/docker-compose.yml (hunk 4 above)
Comment (gurkanindibay) [reply nightly/docker-compose.yml:4]: Should we rollback because it is already on master

**Pull request merged**